cylc/flow/cfgspec/suite.py

```diff
@@ -1361,6 +1349,11 @@ def upg(cfg, descr):
     u.obsolete('8.0.0', ['cylc', 'events', 'abort if any task fails'])
     u.obsolete('8.0.0', ['cylc', 'events', 'mail retry delays'])
     u.obsolete('8.0.0', ['cylc', 'disable automatic shutdown'])
+    u.deprecate(
```

Comment:
Since this config key is "for internal use only" perhaps we should have no upgraders/obsetors? Should we just ensure that there are no tests which use the old syntax?

Reply:
You're quite right, there is no reason for us to provide back-compat here.